strands_navfn/package.xml

```diff
@@ -0,0 +1,52 @@
+<package>
+    <name>strands_navfn</name>
+    <version>1.12.2</version>
```

Comment:
ah yeah, i forgot to ask about something related: how do we handle the fact that this is code copied from the move_base repo? should i also remove the CHANGELOG? Add some extra acknoledgments to navfn maintainers that i removed from the package.xml?